praw/models/reddit/submission.py

```diff
@@ -522,6 +522,33 @@ def nsfw(self):
             API_PATH["marknsfw"], data={"id": self.thing.fullname}
         )
 
+    def set_original_content(self):
```

Comment:
I'm wondering about the naming of the two methods: How come they aren't "mark_original_content" and "unmark_original_content"? I think that is what I would have chosen, but this is also a good choice and I am curious about your reasoning.

Reply:
So, the endpoint itself is `set_original_content`, so I based it off of that. Reddit is not very consistent about naming things, so `mark_*` also will work fine. All three tags have different endpoints/names:

* `marknsfw` vs. `unmarknsfw`
* `spoiler` vs. `unspoiler`
* `set_original_content`

I defer to you guys as to the name of the method that makes the most sense and consistency within PRAW itself.

Reply:
That makes sense. Most similar methods don't have a verb at all (like `nsfw`, `sfw`, `spoiler`, and `unspoiler`), so maybe the method should be named just `original_content`. However, there's not an obvious counterpart to that name (`unoriginal_content`??), which means that that may not be a great choice.

Some of these methods take a parameter like `def contest_mode(self, state=True):` that can be set to `False` to do the opposite action. That's certainly an option here, especially since I would guess that marking a submission as OC is much more common than un-marking it OC.

I guess what I'm saying is that I don't think there's an obvious choice. The reasoning for the name `set_original_content` is as good as or better than the reasoning for any other name. I think that the name is up to you, and after discussing it, I am happy with the names you have chosen.

Reply:
Yeah, there's no easy negation of this particular method unfortunately, and I think your reasoning is absolutely right in that the site probably thinks un-OCing something is quite rare in comparison to setting a submission as OC.

Given all this, my personal preference is to keep `set_original_content` for the simple reason that this endpoint is not officially documented as part of the API yet, and thus we should keep the same name as the endpoint for some modicum of consistency.

Reply:
Sounds good to me!